e2e/external_dep/WORKSPACE

```diff
@@ -0,0 +1,44 @@
+workspace(name = "lib_wksp")
+
+# Override http_archive for local testing
+local_repository(
+    name = "aspect_rules_ts",
+    path = "../..",
+)
+
+#---SNIP--- Below here is re-used in the workspace snippet published on releases
```

Comment:
This is only true for the `e2e/workspace`. See the WORKSPACE files in the other examples and most of the comments here can be deleted.

Reply:
Thanks for pointing that out, updated these two `WORKSPACE` files to copy from the `worker` `e2e` test.